src/SettingsServiceProvider.php

```diff
@@ -36,9 +37,15 @@ public function boot()
         $this->setupRoutes($this->app->router);
 
         // only use the Settings package if the Settings table is present in the database
-        if (!\App::runningInConsole() && count(Schema::getColumnListing('settings'))) {
-            // get all settings from the database
-            $settings = Setting::all();
+        $tableExists = Cache::remember('backpack_settings_table_exists', config('backpack.base.cache_time', 60), function () {
```

Comment:
I see you've used ```config('backpack.base.cache_time', 60)```. Is this a new config value you want to introduce? Since this is only used in Settings, I think it's better for it to be in a ```settings.php``` config file, rather than Base.

But I'm wondering - isn't it easier to just use the default cache time in the app? We're using the default cache store, so I think it's intuitive to use the default cache time too. Then we wouldn't need a ```settings``` config file at all 😄

Reply:
Good points, I did add that in my instance but I'll patch it.